azure/pf-azure-context/functions/public/update-pfazurecontext.ps1

```diff
@@ -0,0 +1,91 @@
+$global:_PfAzureContext
+Function Update-PfAzureContext 
+{
+    Param (
+        [Parameter(Mandatory = $false)] 
+        [string] $SubscriptionName,
+        [Parameter(Mandatory = $false)] 
+        [string] $ResourceGroupName,
+        [Parameter(Mandatory = $false)] 
+        [string] $CompanyAbbreviation,
+        [Parameter(Mandatory = $false)] 
+        [string] $GroupAbbreviation,
+        [Parameter(Mandatory = $false)] 
+        [string] $Label,
+        [Parameter(Mandatory = $false)]  
+        [string] $EnvironmentLetter,
+        [Parameter(Mandatory = $false)] 
+        [AzRegion] $AzRegion
+    )
+
+    if(-not [string]::IsNullOrEmpty($SubscriptionName))
```

Comment:
can we pull the repeated if/else into a util method and pass the variable and error msg in?

Reply:
I tried to do this, but didn't know how to pass in the property name as well to to set that dynamically in the function.  Otherwise, not really saving any code if I have to check each property anyway.  If you could mock something up, it would definitely help me out.

Reply:
Think I figured this out now.

Reply:
ok, was gonna say i can pick that up - i think somewhat similar to what needs to be done for treating object like hashtable